samples/extensions/qa_not_informed_fields_logical_types/v2.0.0/extensions.json

```diff
@@ -0,0 +1,51 @@
+{
+    "type": "record",
+    "name": "qa_not_informed_fields_logical_types",
+    "namespace": "com.telefonica.baikal.qa",
```

Comment:
This extension file is not necessary, just remove this folder

Reply:
OK